test-clients/src/main/java/com/hedera/services/yahcli/README.md

```diff
@@ -0,0 +1,95 @@
+# Yahcli v0.0.1
+A command line interface that is able to perform the following actions against a specified network.
+
+1. Account Operations
+    1. Get Balance.
+2. System File Operations
+    1. Download all/specific file.
+    2. Upload a System file.
+3. Fees Operations
+    1. List Basic Transaction and Query fees of all/specific service.
+
+## Getting Started
+### Setting up your environment
+
+Create a folder structure like this.<br>
+![folderStructure](folderStructure.png)
+
+The _config.yml_ should have contents like:
+
+```
+networks:
+   previewnet:
+     nodes:
+       - { id: 0, account: 3, ipv4Addr: 35.231.208.148 }
+```
+
+You can add details of mulitple networks to this config file with appropriate ipv4Addresses.
+
+_SmartContract.bin_ could be any smartContract bytecode that you want calculate transaction and query fees for.
+
+
+### Executing commands
+
+Run the following command:
+```
+docker run -v $(pwd):/launch gcr.io/hedera-registry/yahcli:0.0.1 help
```

Comment:
Before running this, I guess we need to have docker and posrgres installed?

Reply:
Just installing Docker should be enough. Added that into Environment setup steps. Thanks Leo